ui-grid/ui-grid.d.ts

```diff
@@ -601,6 +601,13 @@ declare module uiGrid {
          */
         enableFiltering?: boolean;
         /**
+        * False by default. When enabled, this adds a settings icon in the top right of the grid, 
+        * which floats above the column header. The menu by default gives access to show/hide columns, 
+        * but can be customized to show additional actions.
+        * @default false
+        */
+        enableGridMenu?: boolean;
```

Comment:
@joeskeen Thanks for the review! I'll update the API docs and will submit a PR soon.